src/views/ApartmentApp/components/StaffMenu/index.js

```diff
@@ -53,9 +53,7 @@ const StaffMenu = ({ userProfile, authentication }) => {
     loadAllCurrentApplications();
 
     // Generate string of today's date in ISO format for use in CSV filename
-    let date = new Date();
-    let dateStr = DateTime.fromJSDate(date).toISODate();
-    setDateStr(dateStr);
+    setDateStr(DateTime.now().toISODate());
```

Comment:
You probably don't need the timezone offset in the ISO Date string output:
```suggestion
    setDateStr(DateTime.now().toISODate({ includeOffset: false }));
```

Reply:
Ah yes, thank you, I was trying to figure out how to efficiently format that